### PR TITLE
Feature/15594 pupils not taking check scaffold

### PR DESCRIPTION
**Pull request by pris54**

### CODE REVIEW

File: admin/views/school/pupils-not-taking-check.ejs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <div class="column-two-thirds" id="checkWindowsForm">
+        <h1 class="heading-xlarge" id="heading-xlarge" tabindex="0">Pupils not taking the check</h1>
+
+        <p class="lede">Click on the button below to add reasons for pupils who are not taking the check</p>
+
+        <p>
+            <a class="button button-primary" href="#">Add reason for pupil</a>
+        </p>
+
+        <br />
```
Comment (jon-shipley) [inline admin/views/school/pupils-not-taking-check.ejs]: Why are all these `br`s here?

Comment (pris54) [reply admin/views/school/pupils-not-taking-check.ejs]: @jon-shipley to add space between the button and the back to top link, so to match the prototype. I don't want a new css class just to add space.

Comment (GeorgeChatzigiannis) [inline admin/views/school/pupils-not-taking-check.ejs]: Can we replace brs with css to match the expected behaviour?

Comment (pris54) [reply admin/views/school/pupils-not-taking-check.ejs]: @GeorgeChatzigiannis I really don't want another CSS class just to add space once ever.

**Pull request merged**